sdk/typescript/runtime/main.go

```diff
@@ -278,9 +315,7 @@ func (t *TypescriptSdk) installedSDK(ctr *Container, runtime SupportedTSRuntime)
 	case Node:
 		return ctr.
 			// Enable corepack so we can use yarn v4 which is supposed to be faster than npm or yarn v1.
-			WithExec([]string{"corepack", "enable"}).
-			WithExec([]string{"yarn", "set", "version", "stable"}).
-			WithExec([]string{"yarn", "workspaces", "focus", "--production"}).Directory(ModSourceDirPath)
+			WithExec([]string{"yarn", "install", "--production"}).Directory(ModSourceDirPath)
```

Comment:
Shouldn't `installedSDK` use the same package manager used for the module's? Why is this needed anyway? Isn't the `./sdk` included in `package.json`? Meaning that installing the module's dependencies would also install the SDK. That goes for the lock as well. The lock in the module should have the transitive dependencies pinned, including the ones from the SDK. This is probably using the SDK's own lock file and `node_modules` which is wasteful.

Reply:
I agree that the sdk shouldn't have its own lockfile.  Since it is listed as a dependency to the users module, it would ultimately be the one responsible for installing and pinning the versioned dependencies

Reply:
To be clear, the SDK does need its own lock file but it's for developing the SDK and running CI tasks on it. Not to be used when installed as dependency like in modules. That lock can be excluded when bundling the SDK into the engine.

Reply:
I also think it's possible to remove these lines but I saved that for a follow up, it's more an optimization than a DX change.

Reply:
> I also think it's possible to remove these lines but I saved that for a follow up, it's more an optimization than a DX change.

That doesn't make sense to me. Your PR is refactoring how dependencies are installed. The SDK seems to be installed separately, unnecessarily. Now is a good time to remove that.

Reply:
Yeah I'll do that!

Reply:
Done!